rechunker/types.py

```diff
@@ -8,8 +9,11 @@
     Optional,
     Tuple,
     TypeVar,
+    Union,
 )
 
+from mypy_extensions import NamedArg
```

Comment:
Is this a necessary runtime dependency? Might need to add it to `install_requires` in the `setup.py` / `setup.cfg`.